cmd/config.go

```diff
@@ -275,7 +282,17 @@ func expandAlias(aliasedURL string) (alias string, urlStr string, hostCfg *hostC
 	// Extract alias from the URL.
 	alias, path := url2Alias(aliasedURL)
 
-	if envConfig, ok := os.LookupEnv(mcEnvHostsPrefix + alias); ok {
+	var envConfig string
+	var ok bool
+
+	if envConfig, ok = os.LookupEnv(mcEnvHostPrefix + alias); !ok {
+		envConfig, ok = os.LookupEnv(mcEnvHostsDeprecatedPrefix + alias)
```

Comment:
What do we do if this `LookupEnv` returns `false`? 
If `false`, there is no point calling `expandAliasFromEnv` when `false`.
As I suggested in the above comment, we can return nil `hostCfg`